content/docs/reference/tutorials/kubernetes/tutorial-migrate-nodegroups.md

```diff
@@ -0,0 +1,454 @@
+Kubernetes is great at managing stateless apps because it's straight-forward to
+run, delete, and most importantly, cleanly recovers in the case of failure
+or lack of quorum, as availability is as simple as spinning up another replica.
+
+If you're managing stateful apps, you should consider leveraging
+[StatefulSets][statefulsets], as they provide each Pod in the set with a unique
+identity and index to employ a systematic approach and ordering to the app's
+lifecycle. With that said, Kuberenetes has no insight into the details of your
+workload, nor how it operates - this is purposefully done by design.
+The work to fully run and operate stateful apps in Kubernetes ultimately lies
+with the app developers, operators, and administrators to bake-in and
+fine-tune the necessary resilience into the workload and its dependencies
+to safely apply updates, and terminate as expected.
+
+As you'll see shortly, we've taken the liberty of using the various
+capabilities that Kubernetes offers for HA and rolling updates to support
+migration of the NGINX Ingress Controller example.
+
+We highly encourage you to check out the references below to help your
+apps achieve proper Kubernetes and production readiness. 
+
+#### References
+
+- [Deployments][k8s-deployments]
+- [PodDisruptionBudget][k8s-pdb]
+- [Node Selectors & Affinity][k8s-node-affinity]
+- [Pod Affinity & Anti-Affinity][k8s-pod-affinity]
+- [Service Selectors][k8s-service]
+- [Readiness & Liveness Probes][k8s-probes]
+- [Lifecycle Hooks][k8s-hooks]
+- [Termation Grace Period & Graceful Shutdown][k8s-graceful-shutdown]
+- [Pod Termination Lifecycle][pod-lifecycle]
+- [Zero-downtime Deployment in Kubernetes with Jenkins][k8s-jenkins]
+
+## The Great Migration
+
+Upon initial deployment of our workload stack, the `echoserver` will ultimately land
+on the Standard node group we've created, given our lack of specificity
+for where the Kubernetes scheduler should place it. The NGINX Ingress Controller
+instead, specifically targets the `2xlarge` node group given its heftier specs.
+Both scheduling choices were made with the intent of segmenting our workloads
+by use-case, and performance requirements we've identified.
+
+In our [inital update][initial-update] we selected that our EKS 
+control plane run on `v1.13.7` of Kubernetes, that the Standard node group
+use `v1.13.7` of the [EKS-optimized][eks-amis] worker AMIs, and that the
+`2xlarge` node group use `v1.12.7` workers.
+
+Since we do not want our `2xlarge` workers to drift too far apart from the control
+plane's version to cause [unsupported skew][version-skew], and we
+ultimately realize that the `2xlarge` node group may not suffice for a
+large, anticipated request load, we've decided to update various settings of 
+the node group for NGINX.
+
+The node group that NGINX will select and target will go from:
+
+  * Using `v1.12.7`of Kubernetes, in a pool of (3) `t3.2xlarge` worker instances ->
```

Comment:
If this `->` is meant to indicate from point A to B, these probably shouldn't be list items?

Reply:
Done